README.md

```diff
@@ -412,7 +412,7 @@ es). It should look like this:
 
 ```yaml
 form-flow:
-  inputs: 'org.formflowstartertemplate.app.inputs.'
+  inputs: 'formflow.library.inputs.'
```

Comment:
This doesn't seem right. It should probably be more like `your.apps.path.inputs` or something like that, right?  This needs to be specific to an implementors path, I think.

Reply:
This is a tricky spot to show an example... not quite sure how to do it.

Reply:
How about:

```suggestion
  inputs: '<your-apps-custom-path-to-inputs>.inputs.'
```

Reply:
I think that would work, it makes it clear that you need to do something there.  I'm not sure there's a good way to do this, so I'd be fine with what you've crafted.